[2021-12-28 19:39:51,957] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: cleansing_dag.cleanse_vision 2021-12-27T19:39:50.349164+00:00 [queued]>
[2021-12-28 19:39:51,970] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: cleansing_dag.cleanse_vision 2021-12-27T19:39:50.349164+00:00 [queued]>
[2021-12-28 19:39:51,971] {taskinstance.py:1094} INFO - 
--------------------------------------------------------------------------------
[2021-12-28 19:39:51,972] {taskinstance.py:1095} INFO - Starting attempt 1 of 2
[2021-12-28 19:39:51,972] {taskinstance.py:1096} INFO - 
--------------------------------------------------------------------------------
[2021-12-28 19:39:51,991] {taskinstance.py:1114} INFO - Executing <Task(PythonOperator): cleanse_vision> on 2021-12-27T19:39:50.349164+00:00
[2021-12-28 19:39:51,996] {standard_task_runner.py:52} INFO - Started process 6762 to run task
[2021-12-28 19:39:51,999] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'cleansing_dag', 'cleanse_vision', '2021-12-27T19:39:50.349164+00:00', '--job-id', '209', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/kym_cleansing.py', '--cfg-path', '/tmp/tmp2oika3d8', '--error-file', '/tmp/tmpux3h7nzu']
[2021-12-28 19:39:52,000] {standard_task_runner.py:77} INFO - Job 209: Subtask cleanse_vision
[2021-12-28 19:39:52,042] {logging_mixin.py:109} INFO - Running <TaskInstance: cleansing_dag.cleanse_vision 2021-12-27T19:39:50.349164+00:00 [running]> on host 0b3e0a2afa7a
[2021-12-28 19:39:52,100] {taskinstance.py:1251} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=cleansing_dag
AIRFLOW_CTX_TASK_ID=cleanse_vision
AIRFLOW_CTX_EXECUTION_DATE=2021-12-27T19:39:50.349164+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-27T19:39:50.349164+00:00
[2021-12-28 19:41:16,031] {python.py:151} INFO - Done. Returned value was: None
[2021-12-28 19:41:16,054] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=cleansing_dag, task_id=cleanse_vision, execution_date=20211227T193950, start_date=20211228T193951, end_date=20211228T194116
[2021-12-28 19:41:16,111] {local_task_job.py:151} INFO - Task exited with return code 0
